src/processors/universal.ts

```diff
@@ -370,6 +360,108 @@ class UniversalProcessor extends BaseOrderProcessor {
     return unlockTx;
   }
 
+  private async unlockOrder(
+    orderId: string,
+    order: OrderData,
+    context: OrderProcessorContext
+  ) {
+    if (this.isUnlockBatchLocked) {
+      setTimeout(() => {
```

Comment:
Looks VERY unreasonable. If the processor is inside of the batch-unlock process, we just need to append this orderId to the `unlockBatchesOrderIdMap`